packages/console/src/pages/Dashboard/components/Skeleton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Card from '@/components/Card';
+
+import * as styles from './index.module.scss';
+
+const Skeleton = () => (
+  <div>
+    <div className={styles.blocks}>
+      {Array.from({ length: 3 }).map((_, index) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
You can use
```
[...Array.from().keys].map()
```
to avoid the index-key warning

Reply:
updated